cropro.py

```diff
@@ -339,6 +377,8 @@ def reject(self):
 def addMenuItem():
     a = QAction(mw)
     a.setText('Cross Profile Search and Import')
+    if SHORTCUT != '':
+        a.setShortcut(SHORTCUT)
```

Comment:
What is the default value? I'd assume its an empty string - if that's the case, then the `if` statement is redundant